tests/tf-manifests/rhcs/clusters/rosa-classic/main.tf

```diff
@@ -93,6 +95,8 @@ resource "rhcs_cluster_rosa_classic" "rosa_sts_cluster" {
   aws_additional_control_plane_security_group_ids = var.additional_control_plane_security_groups
   destroy_timeout                                 = 120
   upgrade_acknowledgements_for                    = var.upgrade_acknowledgements_for
+  base_dns_domain                                 = local.base_dns_domain
+  private_hosted_zone                             = local.private_hosted_zone
```

Comment:
I would say that you don't need to use local but directly vars

Reply:
Addressed.